spec/features/equipment_model_views_spec.rb

```diff
@@ -8,4 +8,56 @@
     it { is_expected.to have_content('Equipment Models') }
     it { is_expected.to have_content(@eq_model.name) }
   end
+  # Testing for issue 1199
+  context 'check for super user' do
+    before do
+      sign_in_as_user(@superuser)
+      visit equipment_models_path
+    end
+    after { sign_out }
+    it { is_expected.to have_content('Reserved') }
```

Comment:
I feel like we should use the same test for admins and superusers as for patrons and checkout persons (`should have_selector('th', text: 'Available'`) rather than this one, for consistency. If you want to go crazy we could technically turn this into a shared example, but for a one-liner it's not strictly necessary (although would be DRY-er).